src/common/copilot/PowerPagesCopilot.ts

```diff
@@ -140,16 +140,21 @@ export class PowerPagesCopilot implements vscode.WebviewViewProvider {
     const pacOutput = await this._pacWrapper?.activeOrg();
 
     if (pacOutput && pacOutput.Status === PAC_SUCCESS) {
-      this.handleOrgChangeSuccess(pacOutput.Results);
+      await this.handleOrgChangeSuccess(pacOutput.Results);
     } else if (!IS_DESKTOP && orgID && activeOrgUrl) {
-      this.handleOrgChangeSuccess({ OrgId: orgID, UserId: userID, OrgUrl: activeOrgUrl } as ActiveOrgOutput);
+      await this.handleOrgChangeSuccess({ OrgId: orgID, UserId: userID, OrgUrl: activeOrgUrl } as ActiveOrgOutput);
     }
 
     webviewView.webview.html = this._getHtmlForWebview(webviewView.webview);
 
     webviewView.webview.onDidReceiveMessage(async (data) => {
       switch (data.type) {
         case "webViewLoaded": {
+          if (this.aibEndpoint === COPILOT_UNAVAILABLE) {
+            this.sendMessageToWebview({ type: 'Unavailable' });
```

Comment:
@tyaginidhi can you please add telemetry also for this: sendTelemetryEvent(this.telemetry, { eventName: CopilotNotAvailable, copilotSessionId: sessionID, orgId: orgID });